src/main/scala/io/iohk/ethereum/crypto/EcKeyGen.scala

```diff
@@ -3,19 +3,24 @@ package io.iohk.ethereum.crypto
 import io.iohk.ethereum.nodebuilder.SecureRandomBuilder
 
 /**
-  * A simple tool to generate and ECDSA key pair. The key pair will be printed in the format:
+  * A simple tool to generate ECDSA key pairs. Takes an optional positional argument [n] - number of key pairs
+  * to generate (default is 1).
+  * The key pairs will be printed in the format:
   *   priv-key-hex (32 bytes)
   *   pub-key-hex (64 bytes)
   *
   * Run:
-  *   ./eckeygen > mantis-datadir/node.key
+  *   ./eckeygen [n] > mantis-datadir/node.key
   *
   * to generate the private key for the node. Note that only the private key will be read upon Mantis boot,
```

Comment:
(minor "Note that only the *first* private key will be read upon Mantis boot"

Reply:
actually it is more general as even if we generate one pair, only the private key is read at boot time and then whole key pair is regenerated based on this key `keyPairFromPrvKey(Hex.decode(privHex))`